pkg/config/tlscfg/cert_watcher_test.go

```diff
@@ -204,7 +199,7 @@ func TestReload_err_watch(t *testing.T) {
 	opts := Options{
 		CAPath: "doesnotexists",
 	}
-	watcher, err := newCertWatcher(opts, zap.NewNop())
+	watcher, err := newCertWatcher(opts, nil, nil, nil)
```

Comment:
It's safer to pass a logger in case future implementation tries to use it before returning the error